apiserver/pkg/util/job.go

```diff
@@ -0,0 +1,63 @@
+package util
+
+import (
+	api "github.com/ray-project/kuberay/proto/go_client"
+	rayalphaapi "github.com/ray-project/kuberay/ray-operator/apis/ray/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type RayJob struct {
+	*rayalphaapi.RayJob
+}
+
+const rayJobDefaultVersion = "1.13"
+
+// NewRayJob creates a RayJob.
+func NewRayJob(apiJob *api.RayJob, computeTemplateMap map[string]*api.ComputeTemplate) *RayJob {
+	var clusterSpec rayalphaapi.RayClusterSpec
+
+	if apiJob.ClusterSpec != nil && len(apiJob.ClusterSelector) == 0 {
+		clusterSpec = *buildRayClusterSpec(rayJobDefaultVersion, apiJob.ClusterSpec, computeTemplateMap)
+	}
+
+	rayJob := &rayalphaapi.RayJob{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:        apiJob.Name,
+			Namespace:   apiJob.Namespace,
+			Labels:      buildRayJobLabels(apiJob),
+			Annotations: buildRayJobAnnotations(apiJob),
+		},
+		Spec: rayalphaapi.RayJobSpec{
+			Entrypoint: apiJob.Entrypoint,
+			Metadata:  apiJob.Metadata,
+			RuntimeEnv: apiJob.RuntimeEnv,
```

Comment:
rayalphaapi.RayJobSpec.RuntimeEnv is base64 string, do we want to expose it same way as base64 to user in apiserver or just ask user to type json string and we can encode it?

Reply:
added, now user could write the runtime env like this:
```yaml
"runtimeEnv": "{'pip': ['requests==2.26.0','pendulum==2.1.2'],'env_vars': {'counter_name': 'test_counter'}}'",
```